src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/activitiesappointments/ActivitiesAppointmentsClientTest.kt

```diff
@@ -50,47 +47,6 @@ class ActivitiesAppointmentsClientTest {
     )
   }
 
-  @Test
-  fun `should get single prisoners appointment`() {
-    server.stubGetPrisonersAppointments(
-      prisonCode = BIRMINGHAM,
-      prisonerNumber = "123456",
-      date = tomorrow(),
-      locationIds = setOf(1000, 2000),
-    )
-
-    val appointment = client.getPrisonersAppointmentsAtLocations(BIRMINGHAM, "123456", tomorrow(), 1000).single()
-
-    appointment.internalLocation?.id isEqualTo 1000
-  }
-
-  @Test
-  fun `should get multiple prisoner appointments at locations`() {
-    server.stubGetPrisonersAppointments(
-      prisonCode = BIRMINGHAM,
-      prisonerNumber = "123456",
-      date = tomorrow(),
-      locationIds = setOf(1000, 2000, 3000),
-    )
```

Comment:
This would suggest a usage for multiple locations?

Reply:
Yes but it was never used that way.